packages/devtools_app/lib/src/performance/flutter_frames_chart.dart

```diff
@@ -315,13 +319,23 @@ class FlutterFramesChartItem extends StatelessWidget {
             color: defaultSelectionColor,
             height: selectedIndicatorHeight,
           ),
+        if (frame.hasShaderTime)
```

Comment:
use a minimum threshold for the amount of shader time (maybe 4 or 8ms) and only warn if the frame missed the raster time target.

Reply:
done.